src/main/java/frc/robot/turret/TurretSubsystem.java

```diff
@@ -71,18 +83,17 @@ private void handleState(TurretState state) {
 
     @Override
     protected String getLogPath() {
-        return null;
+        return LOGGING_PATH;
```

Comment:
```suggestion
        return TurrentConstants.LOG_PATH;
```
must of the code is in this convention and we DON'T BREAK CONVENTIONS

Reply:
just for you to know. i use a static import for the `TurrentConstants.LOG_PATH`